Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -118,9 +119,16 @@ var TouchableNativeFeedback = React.createClass({
      *
      * @param color The ripple color
      * @param borderless If the ripple can render outside it's bounds
+     * @param borderWithRadius If the ripple should be contained with the target's border-radius, if defined. Only
+     * used if {@code borderless} is false.
      */
-    Ripple: function(color: string, borderless: boolean) {
-      return {type: 'RippleAndroid', color: processColor(color), borderless: borderless};
+    Ripple: function(color: string, borderless: boolean, borderWithRadius: boolean) {
+      return {
+        type: 'RippleAndroid',
+        color: processColor(color),
+        borderless: borderless,
+        borderWithRadius: borderWithRadius
```

Comment:
is there a use case for having a view have a borderRadius, but having `Ripple` not constrain itself to it?  I'm wondering if we can just have this always happen?